scripts/wildfly/install-wildfly.sh

```diff
@@ -21,7 +21,7 @@ export PATH=$PATH:$(pwd)/galleon/bin
 
 echo ">>> Installing WildFly ..."
 rm -fr $WILDFLY_SERVER_DIR
-galleon.sh install wildfly#$WILDFLY_VERSION --dir=$WILDFLY_SERVER_DIR --layers=jaxrs-server,microprofile-health,microprofile-fault-tolerance,elytron-oidc-client,metrics,opentelemetry
+galleon.sh install wildfly#$WILDFLY_VERSION --dir=$WILDFLY_SERVER_DIR --layers=elytron-oidc-client,jaxrs-server,mail,metrics,microprofile-health,microprofile-fault-tolerance,opentelemetry
```

Comment:
I suggest we pull these layers from the pom file, like so (thanks for the script CoPilot :)
```
export WILDFLY_LAYERS=$(awk -F'[<>]' '/<layer>/{printf "%s,", $3}' pom.xml | sed 's/,$//')
```
and use that value, so that we don't have to keep the layers in synch manually !